hw/dv/sv/alert_agent/alert_agent.core

```diff
@@ -16,15 +16,21 @@ filesets:
       - alert_agent_cfg.sv: {is_include_file: true}
```

Comment:
apologies for requesting this change - since there is alert_esc_base_driver to indicate its a 'common' type, I'd prefer following the same with other files too {alert_esc_agent_cfg, alert_esc_agent_pkg, alert_esc_agent, alert_esc_if etc.}. That makes it clear that it perform double duty as alert as well as escalation agent.